app/views/downloads/download_linux.html.erb

```diff
@@ -12,49 +12,74 @@
+  <p>Replace <i>file.rpm</i> with the path to the RPM file that you installed.</p>
 
   <h3>Nix/NixOS</h3>
-  <code># nix-env -i git</code>
+  <pre style="height: 10px;overflow-y: hidden;padding-top: 0;margin-top: 0;"># nix-env -i git</pre>
+  <p>To install gitk:</p>
+  <pre style="height: 10px;overflow-y: hidden;padding-top: 0;margin-top: 0;"># nix-env -iA nixos.gitAndTools.gitFull</pre>
 
   <h3>FreeBSD</h3>
-  <code># pkg install git</code>
+  <pre style="height: 10px;overflow-y: hidden;padding-top: 0;margin-top: 0;"># pkg install git</pre>
   
   <h3>Solaris 9/10/11 (<a href="https://www.opencsw.org">OpenCSW</a>)</h3>
-  <code># pkgutil -i git</code>
+  <pre style="height: 10px;overflow-y: hidden;padding-top: 0;margin-top: 0;"># pkgutil -i git</pre>
+  <p>Optionally, if you would also like to install gitk (on <a href="https://www.opencsw.org/packages/gitk/#:~:text=Solaris%2010%20and%2011">Solaris 9/10</a> only), then run the following:</p>
+  <pre style="height: 10px;overflow-y: hidden;padding-top: 0;margin-top: 0;"># pkgadd -d http://get.opencsw.org/now</pre>
+  <pre style="height: 10px;overflow-y: hidden;padding-top: 0;margin-top: 0;"># /opt/csw/bin/pkgutil -U</pre>
+  <pre style="height: 10px;overflow-y: hidden;padding-top: 0;margin-top: 0;"># /opt/csw/bin/pkgutil -y -i gitk </pre>
+  <pre style="height: 10px;overflow-y: hidden;padding-top: 0;margin-top: 0;"># /usr/sbin/pkgchk -L CSWgitk # list files</pre>
 
   <h3>Solaris 11 Express</h3>
-  <code># pkg install developer/versioning/git</code>
+  <pre style="height: 10px;overflow-y: hidden;padding-top: 0;margin-top: 0;"># pkg install developer/versioning/git</pre>
   
   <h3>OpenBSD</h3>
-  <code># pkg_add git</code>
+  <pre style="height: 10px;overflow-y: hidden;padding-top: 0;margin-top: 0;"># pkg_add git</pre>
 
   <h3>Alpine</h3>
-  <code>$ apk add git</code>
+  <pre style="height: 10px;overflow-y: hidden;padding-top: 0;margin-top: 0;"># apk add git</pre>
+  <p>Gitk is a subpackage of git on Alphine Linux. It can be invoked with git-gitk.</p>
   
   <h3>Red Hat Enterprise Linux, Oracle Linux, CentOS, Scientific Linux, et al.</h3>
   <p>RHEL and derivatives typically ship older versions of git. You can <a href="https://www.kernel.org/pub/software/scm/git/">download a tarball</a> and build from source, or use a 3rd-party repository such as <a href="https://ius.io/">the IUS Community Project</a> to obtain a more recent version of git.</p>
 
   <h3>Slitaz</h3>
-  <code>$ tazpkg get-install git</code>
+  <pre style="height: 10px;overflow-y: hidden;padding-top: 0;margin-top: 0;"># tazpkg get-install git</pre>
+  <p>If you also want to install gitk click <a href="http://forum.slitaz.org/topic/gitk?bb_attachments=43156&bbat=2350">here</a> to get a tazpkg file for gitk. After installing it, run:</p>
+  <pre style="height: 10px;overflow-y: hidden;padding-top: 0;margin-top: 0;"># tazpkg install file.tazpkg</pre>
```

Comment:
same here: not sure if we want to add more text just to explain how to install from a tazpkg file.

also, we are linking to a gitk from 3 years ago. if there isn't an up to date version maybe we shouldn't link? Seems odd for the "oficial" git website to link to 3rd party executables that were not verified

Reply:
I don't know. He has given over 1000 hours of support in the community and that attachment was downloaded hundreds of times without any complaints in the last years. Nowhere else does anyone mention the word ```gitk``` for Slitaz. If you want to, we can remove the instructions for Slitaz.

Reply:
I'd prefer to err on the side of _not_ including information if we're not 100% sure it's good advice. People who use individual platforms can search themselves, too, and I don't want to imply endorsement if we're not sure the package is up to date and none of us has actually used it.

Reply:
Sure I will make a commit removing ```Slitaz``` instructions.